pkg/kind/cluster.go

```diff
@@ -90,6 +90,10 @@ func (c *Cluster) Reconcile(ctx context.Context, recreate bool) error {
 		return err
 	}
 
+	fmt.Print("########################### Our kind config ############################\n")
```

Comment:
Any objections to using a logger? I know it uses `fmt` here and other places, but I feel like it's nicer to consolidate logging stuff to one way.  We currently use mix of fmt and logger and we should consolidate to one.

Reply:
I agree with you. concerning this point, I commented this ticket #15 to propose a better way to log.
We will fix that in a separate PR